genon/preprocessor/tests/unit/test_hwp_routing_unit.py

```
@@ -0,0 +1,143 @@
+async def test_non_hwp_extension_does_not_route_to_hwp_processor(attachment_processor, ext):
+    """
+    HWP/HWPX가 아닌 확장자는 hwp_processor로 가면 안 된다.
+    """
+    dp = attachment_processor()
+    dp.hwp_processor = AsyncMock(return_value=[])
+
+    fake_request = MagicMock()
+    fake_request.is_disconnected = AsyncMock(return_value=False)
+
+    try:
+        await dp(fake_request, f"/tmp/test_file{ext}")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_file"

(S108)